### PR TITLE
split topBar from Main component

### PR DESCRIPTION
**Pull request by hkcj**

### CODE REVIEW

File: src/admin/moudles/common/components/TopBar.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+
+import { Typography, Badge } from '@material-ui/core';
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+// import MenuIcon from '@material-ui/icons/MenuIcon';
+import { makeStyles } from '@material-ui/core/styles';
+
```
Comment (zt1983811) [inline src/admin/moudles/common/components/TopBar.tsx]: Nitpick: can we remove the break line please?

File: src/admin/moudles/common/components/TopBar.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import Toolbar from '@material-ui/core/Toolbar';
+import clsx from 'clsx';
+import React, { useContext } from "react";
+import { drawerWidth } from "../../Main";
+import { ToggleContext } from "../providers/ToggleProvider";
+
```
Comment (zt1983811) [inline src/admin/moudles/common/components/TopBar.tsx]: Nitpick: can we remove the break line please?

Comment (zt1983811) [inline src/admin/moudles/common/components/TopBar.tsx]: Nitpick: can we remove the break line please?

**Comment by zt1983811** [comment]
@hkcj  Could you fix what I comment above so that we can merge this one quick quick please? Thanks

**Comment by zt1983811** [comment]
LGTM let's get it merge @hkcj

**Comment by hkcj** [comment]
ok, here goes nothing...

**Pull request merged**